elementary/clients/dbt/dbt_log.py

```diff
@@ -0,0 +1,38 @@
+
+from dataclasses import dataclass
+import json
+from typing import Iterator, Optional
+
+from elementary.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class DbtLog:
+    msg: Optional[str]
+    level: Optional[str]
+    exception: Optional[str]
```

Comment:
I'd personally call it `err_msg`, but not critical.

Reply:
i like it better in the code as `exception` seeing as its name in dbt is `exc`, but i will change the `__str__` usage to Error